src/components/svgImage/svgImage.api.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "SvgImage",
```

Comment:
SvgImage is an internal component - displayName = IGNORE and it's not exported in the src/index file. Please remove all api json files for internal components